src/model/layer.ts

```diff
@@ -268,10 +268,10 @@ export class ColumnModel extends BaseArrowLayerModel {
   protected radius: GeoArrowColumnLayerProps["radius"] | null;
   protected angle: GeoArrowColumnLayerProps["angle"] | null;
 
-  // @ts-expect-error Property 'vertices' has no initializer and is not
+  // // @ts-expect-error Property 'vertices' has no initializer and is not
```

Comment:
@kylebarron should we clean up these commented lines?

Reply:
I'll add a comment here that the commented out lines represent deck.gl properties that we don't currently expose to Python